common/src/main/java/io/netty/util/concurrent/DefaultThreadFactory.java

```diff
@@ -120,7 +126,7 @@ public Thread newThread(Runnable r) {
     }
 
     protected Thread newThread(Runnable r, String name) {
```

Comment:
@zhangkun83 could you add a "TODO:" here? Something like:

`// TODO: Once we can break the API we should add ThreadGroup to the arguments of this method.`